src/server/create/helpers/audit-created.js

```diff
@@ -0,0 +1,7 @@
+async function auditCreated(request, repositoryKind, repositoryName) {
```

Comment:
TBH not sure if we need this abstraction of an abstraction?

Reply:
There were a lot of duplicate code all over without it. With Strings. Standardising on the template somehow seems neater.